draft-ietf-httpbis-message-signatures.md

```diff
@@ -26,7 +26,7 @@ author:
     region: WA
     code: 98108-1226
     country: United States of America
-
+    role: editor
```

Comment:
assume this means you can merge PRs? : )

Reply:
No, this is just a designation within the IETF -- often documents have multiple "authors" but a single designated "editor" to lead the unruly crew. I noticed @richanna 's entry didn't have that flag set so I grabbed that change on the way by since we're updating the framing text.